DuosForSums.java

```diff
@@ -0,0 +1,49 @@
+import java.util.Arrays;
+
+public class DuosForSums {
+
+    public static void main(String[] args) {
+        int[] initArr = {1, 3, 5, -2, 6, 0, -3, -1, 7, 2};
+        int sumNum = 5;
+        group(initArr, sumNum);

```

Comment:
What if _initArr = {5, 5, 5, 5, 5, 5, 5, 5, 5, 5}; sumNum = 10;_? Will it show all pairs (45)

Reply:
Only unique pairs  [[5, 5], [5, 5], [5, 5], [5, 5], [5, 5]]

Reply:
How many unique pairs has my example?)

Reply:
unique means, if a number is already used then it is out and not used anymore